packages/schema/src/config/typescript.ts

```diff
@@ -50,8 +50,7 @@ export default defineUntypedSchema({
     /**
      * Generate a `*.vue` shim.
      *
-     * We recommend instead either enabling [**Take Over Mode**](https://vuejs.org/guide/typescript/overview.html#volar-takeover-mode) or adding
-     * TypeScript Vue Plugin (Volar)** 👉 [[Download](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin)].
+     * We recommend instead installing the [official Vue extension](https://marketplace.visualstudio.com/items?itemName=Vue.volar).
      */
     shim: true
```

Comment:
We should update the default to false in this case.

Reply:
yeah for sure